### PR TITLE
fix: use OS images for artblocks for faster loading

### PR DESCRIPTION
**Pull request by tv3636**

### CODE REVIEW

File: src/custom/artblocks-engine/index.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const fetchCollection = async (_chainId, { contract, tokenId }) => {
 export const fetchToken = async (_chainId, { contract, tokenId }) => {
   const url = `https://token.artblocks.io/${contract}/${tokenId}`;
   const { data } = await axios.get(url);
+  const osData = await opensea.fetchTokens(_chainId, [{ contract, tokenId }]);
```
Comment (nofir) [inline src/custom/artblocks-engine/index.js:49]: you need to have a try catch here and if it fails, fallback to the artblocks image.

File: src/custom/artblocks/index.js [hunk 2]
```diff
@@ -73,8 +74,9 @@ export const fetchToken = async (_chainId, { contract, tokenId }) => {
     tokenId,
     collection: _.toLower(`${contract}:${startTokenId}:${endTokenId}`),
     name: data.name,
-    imageUrl: data.image,
+    imageUrl: osData[0].imageUrl,
```
Comment (nofir) [inline src/custom/artblocks/index.js:77]: similar as above

File: src/custom/artblocks-engine/index.js (hunk 1 above)
Comment (nofir) [reply src/custom/artblocks-engine/index.js:49]: Also, look at line 33 (for getting collection metadata), we are already getting the opensea metadata, might make sense to also use the opensea image for the collection (not sure if its actually an issue there or not)

**Review by nofir — CHANGES_REQUESTED** [review]

File: src/custom/artblocks-engine/index.js (hunk 1 above)
Comment (tv3636) [reply src/custom/artblocks-engine/index.js]: added the try/catch to both. 

for the collection image here we use the image associated with the tokenId requested - replacing with the OS response returns a different image, so i left that out for now

**Pull request closed without merging**